install/resources/settings.xml

```diff
@@ -959,7 +959,8 @@ disabled=Disable this feature]]></optionscode>
 		</setting>
 		<setting name="username_method">
 			<title>Allowed Login Methods</title>
-			<description><![CDATA[The login methods you wish to allow for the username field. Username only, Email only, or allow both.]]></description>
+			<description><![CDATA[The login methods you wish to allow for the username field. Username only, Email only, or allow both.<br />
+<strong>Note:</strong> This will only work for the account that registered the e-mail first if the "Allow emails to be registered multiple times?" setting was set to "Yes" before changing this.]]></description>
```

Comment:
The note is not up-to-date anymore, is it? The setting is not allowed to be changed when there are duplicates.